templates/kickstart.j2

```diff
@@ -43,8 +60,9 @@ ignoredisk --only-use={{ hostvars[groups[item][0]]['os_disks'] }}
 {{ line }}
 {% endfor %}
 
-{% if hostvars[groups[item][0]]['dhcp_kickstart_manage_packages'] is not defined %}
 %packages
+{% if [groups[item][0]]['dhcp_kickstart_centos_version'] is "7" %}
```

Comment:
should we have "@core" as a default for centos8? Now if I read it correctly, it has to be defined in the group

Reply:
This would be a good addition.

But I'm ok with this as long as it's documented in the Readme.